recipe/conda_build_config.yaml

```diff
@@ -8,4 +8,7 @@ lmdb:
   - "0.9.22"
 folly:
   # This version of folly is used to make the build usable by more environments
-  - "2023.09.25.00"	
+  - "2023.09.25.00"
+libprotobuf:
+  # This version of libprotobuf is used to make the build ubasable by BQNT
+  - "3.21.12"
```

Comment:
The re-rendering has not happened, which is slightly weird.

Could you try to pin this version of `libprotobuf` in `recipe/meta.yaml`?